Doc/library/importlib.rst

```diff
@@ -1337,8 +1338,9 @@ find and load modules.
 
    (``__package__``)
 
-   (Read-only) Fully-qualified name of the package to which the module
-   belongs as a submodule (or ``None``).
+   (Read-only) The fully-qualified name of the package under which the module
+   should be loaded as a submodule (or the empty string for top-level modules).
```

Comment:
```suggestion
   should be loaded as a submodule.
```

Reply:
Similarly, I think it's fine to mention the empty string here too.